config_example.sh

```diff
@@ -2,7 +2,14 @@
 
 # You can get this secret from https://cloud.redhat.com/openshift/install/pull-secret
 set +x
-export PULL_SECRET=''
+export PERSONAL_SECRET=''
```

Comment:
Since we're changing this anyway, it would be preferable if this were just a path to a file. Keeping it inline makes the config file hard to edit, and complicates stuff like using Ansible to set up a host. I realise people can just do `PERSONAL_SECRET="$(cat secret.json)"`, but there's never any real reason for it to be inline. We could even provide a default file name and ignore it in .gitignore. Would certainly avoid all the +x/-x stuff as well.

Reply:
+1 This will help also in the CI builds an minimize the chance of a leak

Reply:
The CI_TOKEN value is logged when it is used in the command to login to get the pull secret. I'm not sure we can avoid that, but since we're only going to use the tokens for personal users maybe it doesn't matter?

Reply:
I don't expect to use CI_TOKEN it in the CI builds, since we already have a combined pull secret file prepared by the CI lower infrastructure (ci-operator), see https://github.com/openshift/release/blob/6bbee5476c9a8487fea4d12d227e8e52f480e998/ci-operator/step-registry/baremetalds/devscripts/setup/baremetalds-devscripts-setup-commands.sh#L78

Reply:
OK, good. I think the latest version of this is probably logging fewer secret values, then.

Reply:
config_example is out of date wrt the code now.